Adds instructions to deploy locally without Cloud Build. (#105)
amandakarina/terraform-example-foundation: 1-org/README.md

### Running Terraform locally

1. The next instructions assume that you are at the same level of the `terraform-example-foundation` folder.
Create `gcp-org` folder, copy `1-org` content and Terraform wrapper script; ensure it can be executed.

   ```bash
   mkdir gcp-org
   cp -R terraform-example-foundation/1-org/* gcp-org/
   cp terraform-example-foundation//build/tf-wrapper.sh gcp-org/
   cp terraform-example-foundation/.gitignore gcp-org
   cd gcp-org
   chmod 755 ./tf-wrapper.sh
   ```

> 1. Initialize a local git repository, so you can manage versions locally. Create the environment branches.

Reviewer: ```suggestion
1. Initialize a local Git repository to manage versions locally. Then, create the environment branches.
```
